pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -85,6 +85,11 @@ func (p *pvcEvaluator) Constraints(required []api.ResourceName, item runtime.Obj
 	return nil
 }
 
+func (p *pvcEvaluator) GetPriorityClass(item runtime.Object) (*string, error) {
```

Comment:
its not clear to me yet why GetPriorityClass needs to be defined in each evaluator yet rather than just scoped to the pod evaluator.